### ISSUE TITLE
Card validation method does not work properly

### ISSUE THREAD

**Issue by petrnohejl** [balanced/balanced-android#8]
Method Card.isValid() does not work:
- If I set expiration month to 13, it returns true
- If I set expiration year to some past value, e.g. 2012, it returns true

**Comment by petrnohejl** [comment]
Method [Card.isExpired()](https://github.com/balanced/balanced-android/blob/master/balanced-android/src/main/java/com/balancedpayments/android/Card.java#L64) seems to be wrong.

This code makes no sense:

```
if (expirationMonth > 12 || expirationYear < 1) { return false; }
```

Probably should be:

```
if(expirationMonth > 12 || expirationMonth < 1) return true; // true means invalid/expired
```

Btw, second line in following code is useless, because Calendar.getInstance() returns current date.

```
Calendar cal = Calendar.getInstance();
cal.setTime(new Date()); // this is useless
```

**Comment by remear** [comment]
@petrnohejl Yes. it should return true. Would you like to open a pull request to make these changes?
